posthog/migrations/0146_eventproperty_sync.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.11 on 2021-04-05 19:23
+
+from django.db import migrations
+
+
+def sync_team_event_names_and_properties(apps, schema_editor):
+    Team = apps.get_model("posthog", "Team")
+    for team in Team.objects.all():
+        try:
+            # importing locally in case the code ever moves
+            from posthog.tasks.sync_event_and_properties_definitions import sync_event_and_properties_definitions
+
+            sync_event_and_properties_definitions.delay(team.uuid)
```

Comment:
Will celery workers have restarted at this point and be able to exec this?

Reply:
Hmmm yeah actually I think it's better to run this synchronously

Reply:
Good point. We can't guarantee that, so removed.